### PR TITLE
Expansion2 merges

### PR DESCRIPTION
**Pull request by joelekstrom**
I'll try to help a bit with the merging, please check if this is the correct way to do it (ignoring any new alignment changes). I had some questions regarding this in https://github.com/Azurency/CQUI_Community-Edition/issues/380#issuecomment-466543885

I'll also be careful not to duplicate any work from https://github.com/Azurency/CQUI_Community-Edition/pull/382

### CODE REVIEW

File: Assets/UI/worldviewiconsmanager.lua [hunk 1]
```diff
@@ -757,11 +772,11 @@ function Initialize()
   Events.ResourceVisibilityChanged.Add(OnResourceVisibilityChanged);
   Events.ResourceAddedToMap.Add(OnResourceChanged);
   Events.ResourceRemovedFromMap.Add(OnResourceRemovedFromMap);
-  Events.ImprovementAddedToMap.Add( OnResourceChanged );
```
Comment (Azurency) [inline Assets/UI/worldviewiconsmanager.lua:760]: Should not be removed, it's a CQUI addition, in fact event add 
```Events.ImprovementAddedToMap.Add( OnResourceChanged ); -- CQUI : modified to emphasizing resources that need improvement```
You can see that in commit https://github.com/Azurency/CQUI_Community-Edition/commit/4d718a7801494fe7c61fe35cb35ed7b85cdb94d0

File: Assets/UI/worldviewiconsmanager.lua [hunk 2]
```diff
@@ -645,8 +664,6 @@ end
 
 -- ===========================================================================
 function AddImprovementRecommendationsForCity( pCity:table, pSelectedUnit:table )
-  do return end;
```
Comment (Azurency) [inline Assets/UI/worldviewiconsmanager.lua:648]: Should not be removed https://github.com/Azurency/CQUI_Community-Edition/commit/849da953be8f479eb4c90fa953024c0644313bf8. It's a CQUI addition

**Review by Azurency — COMMENTED** [review]
That's the idea, but be careful. Do not compare the CQUI version and the game version to make the changes. Compare the old game version and the new game version, and then merge theses changes into the CQUI file making sure not to break any CQUI feature, not like you did here.

File: Assets/UI/worldviewiconsmanager.lua (hunk 1 above)
Comment (joelekstrom) [reply Assets/UI/worldviewiconsmanager.lua:760]: I see, thanks so much for the guidance. I will give it another try!

**Comment by Azurency** [comment]
Thanks for trying to help, I'll close this pull request because in the end I'll remove the `worldviewiconsmanager.lua` file from CQUI and use the "new" (last-expansion-old) modding logic that just replace the functions we need.

**Comment by joelekstrom** [comment]
Good call. I still haven’t taken the time to learn this stuff properly. But you know how it is... hopefully tomorrow 😄

**Pull request closed without merging**